proto/proto/store.proto

```diff
@@ -64,44 +66,232 @@ service Rpc {
     // For preserving some degree of privacy, note tags and nullifiers filters contain only high
     // part of hashes. Thus, returned data contains excessive notes and nullifiers, client can make
     // additional filtering of that data on its side.
-    rpc SyncState(requests.SyncStateRequest) returns (responses.SyncStateResponse) {}
+    rpc SyncState(shared.SyncState) returns (shared.SyncedState) {}
 
     // Returns the status info.
-    rpc Status(google.protobuf.Empty) returns (responses.StoreStatusResponse) {}
+    rpc Status(google.protobuf.Empty) returns (shared.StoreStatus) {}
```

Comment:
Similar to the last comment, I would define `StoreStatus` here and then just import it from here wherever it is needed.